_docs/pages/supported-tracers.md

```diff
@@ -6,7 +6,7 @@
 
 ## Jaeger
 
-[Jaeger \\ˈyā-gər\\](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
+[Jaeger \\ˈyā-gər\\](https://github.com/jaegertracing/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
```

Comment:
Looks good, however could you please use `http://jaegertracing.io`? Now it redirects to the repo, in the short future it will point to the new website.

Reply:
There is also a C++ tracer https://github.com/jaegertracing/cpp-client now